telescope/query.py

```diff
@@ -163,13 +163,11 @@ def _create_query_string(self):
                               'FROM\n\t{table}\n'
                               'WHERE\n\t{conditional_list}')
         non_null_fields = ['connection_spec.data_direction',
-                           'web100_log_entry.is_last_entry',
                            'web100_log_entry.snap.HCThruOctetsAcked',
                            'web100_log_entry.snap.CongSignals',
                            'web100_log_entry.connection_spec.remote_ip',
                            'web100_log_entry.connection_spec.local_ip']
-        tool_specific_conditions = ['project = 0',
-                                    'web100_log_entry.is_last_entry = True']
+        tool_specific_conditions = ['project = 0']
```

Comment:
Consider renaming to just like `project_conditions`. `tool_specific_conditions` is a vestige of the days when we thought Telescope would do paris-traceroute queries too.

Or just ditch the variable entirely since we only use it once and put `['project = 0']` in line 177.